packages/SketchMorph2-Core.package/MorphicMonet.class/instance/activateDefaultItem.st

```diff
@@ -0,0 +1,7 @@
+defaults
+activateDefaultItem
+	| defaultItem |
+	defaultItem := (self submenuDict at: 'Brushes') value detect: [:item |
+			(item isM2MenuItemMorph and: [item isDefaultTool])]. "Pencil tool"
+	self menuPanel submorphs third clickItem: defaultItem.
```

Comment:
I don't think it is a good idea to use the third panel and expecting the paint tool selection panel there.

Reply:
I'm not sure if there is another/better way to access it due to the way it is created. (`MorphicMonet>>loadMenu`)

Reply:
Hm. Maybe we could check which submorph contains `defaultItem` and then select it?

Reply:
Yeah that could work.

Reply:
Due to the short time we have remaining we can leave this as it is, in my opinion.